### ISSUE TITLE
Issue during Install

### ISSUE THREAD

**Issue by RichShumaker** [pmario/tweb-config#3]
Mario,

I ran into this issue when I was using the install script.
When I tried to start the twmanager server I received the same error.

pi@raspberrypiserver ~ $ twanager server
Traceback (most recent call last):
  File "/usr/local/bin/twanager", line 19, in <module>
    handle(args)
  File "/usr/local/lib/python2.7/dist-packages/tiddlyweb/manage.py", line 66, in  handle
    from tiddlyweb.config import config
  File "/usr/local/lib/python2.7/dist-packages/tiddlyweb/config.py", line 155, i n <module>
    from httpexceptor import HTTPExceptor
ImportError: No module named httpexceptor

I did a full LAMP install before using your script.

Thank you very much for creating this to help users like me install this on a RasPi

Rich Shumaker

**Comment by pmario** [comment]
The script is 2 years old and TiddlyWeb has changed quite a bit. .. So it is missing a python module named "httpexceptor" ... you can try to install it manually with `pip install httpexceptor` ... There may be more missing modules. 

Do you need a LAMP stack? ... This setup shouldn't need one.

**Comment by RichShumaker** [comment]
d'oh!!!
Thanks Mario and that makes perfect sense.
I need to explore a more simple TW module and I read something this morning you wrote.
I think at the TW Group so I will need to go read that.

All I want is something like FTP or SFTP to post the HTML file to the server.
TW5 would need to be able to store the encrypted password in a tiddler and send it over.
Sounds easy but I know it is much harder than that.

I understand that TW5 could not see local files on the server and I wonder if it would be possible to solve that on the server through permissions.

I did the full install as it was easy and sorta quick, building the software on the Pi has a lot of tutorials and is fairly straight forward.

I use BerryBoot and that has caused problems with things in the past.

Ideally I could load the Berry Web server – Lighttpd webserver with PHP and SQlite for serving websites from your Raspberry Pi.  Then load a simple module for TiddlyWiki 5.
That process of BerryBoot and Lighttpd is only about 10 minutes and you are up and running.
If we can add a simple method for TW5 integration then I could see kids using the RasPi to take notes in TW5.  From there we can see how to expand TW5 to control things on the PI.
Need to get it easily working on a Pi first.
Thanks again Mario.
This is CLOSED.

**Comment by pmario** [comment]
> Ideally I could load the Berry Web server – Lighttpd webserver with PHP and SQlite for serving websites from your Raspberry Pi. Then load a simple module for TiddlyWiki 5.

You are mixing things here. ... The TiddlyWeb server backend, that can store single tiddlers in a store is written in python. ...

python has nothing to do with PHP. If you want to use TW with PHP backend, you'd need someone, who writes a TiddlyWeb API with PHP. 

The problem is, that a web stack is not plug and play. ... The different systems have some dependencies. ...

I'll have a look at tweb-config and send an update here, if I find something new.

**Comment by RichShumaker** [comment]
Thanks Mario
I found the PHP script BidiX created but it didn't work in April of 2014 and I can't get it to work now.
I have now installed Node.js and TiddlyWiki on the Raspberry Pi as this seemed like the best way to go.

From reading about TiddlyWeb it does not work for TW5 and until that is fixed it doesn't make sense to break my neck to try on the Pi as I want TW5 on the Pi.

I get what TiddlyWeb now is and I apologize for my confusion.  It is a Python Program that runs with a Python Server no LAMP needed.
